File: python/7 kyu/419.py
# https://www.codewars.com/kata/57a60bad72292d3e93000a5a/train/python

# Passed

def to_acronym(inp):
    words = inp.split()
    result = str()
    for word in words:
        result += word[0].upper()
        
    return result

output = to_acronym("hyper text markup language")
print(output)

File: nodejs/7 kyu/419.js
// https://www.codewars.com/kata/570f6436b29c708a32000826/train/javascript

// Passed

function firstNonRepeated(s) {
  let charToCountMap = new Object();
  for (const char of s) {
    if (!(char in charToCountMap)) {
      charToCountMap[char] = 0;
    }

    charToCountMap[char]++;
  }

  let result = null;
  for (const char of s) {
    let charCount = charToCountMap[char];
    if (charCount === 1) {
      result = char;
      break;
    }
  }

  return result;
}

const output = firstNonRepeated("test");
console.log(output);